packages/core-dialog/core-dialog.js

```diff
@@ -36,10 +36,11 @@ export default class CoreDialog extends HTMLElement {
         this.backdrop.style.zIndex = zIndex + 1
         active.setAttribute(this._opener, '') // Remember opener element
         setFocus(this)
+        setTimeout(() => setFocus(this)) // Move focus after paint (helps iOS and react portals)
```

Comment:
should we just remove the previous `setFocus(this)` here then? :)

Reply:
the only reason i didn't was that the next call does the same and as such I thought that might have been for a reason... 😊 

https://github.com/nrkno/core-components/blob/ae9f427c9204dd250f2c4b16a9b9b8bb96122f12/packages/core-dialog/core-dialog.js#L39-L42

Also, since `setFocus` first checks if focus is already inside `this`, I thought it wouldn't really hurt?